src/commands/pczt/inspect.rs

```diff
@@ -150,6 +180,7 @@ impl Command {
                                 .and_then(|(derivation, (seed_fp, coin_type))| {
                                     derivation.extract_account_index(seed_fp, *coin_type)
                                 }),
+                            decrypted,
                         )
```

Comment:
This should be a struct that can guide what gets printed, since we're double covering some printed information.